music/src/music_vae/model.ts

```diff
@@ -1403,6 +1403,106 @@ class MusicVAE {
             'Sampling completed', startTime, 'MusicVAE', logging.Level.DEBUG));
     return outputSequences;
   }
+
+  /**
+   * Generates similar tensors to an input tensor.
```

Comment:
I wonder if this should be implemented in a downstream app rather than in the core API. @notwaldorf what do you think?  It seems a bit experimental/specialized.

Reply:
Happy to be convinced!

Reply:
I agree with you, just not sure where to put it.  This functionality was requested by cjcai@

Reply:
@cjcai I mean

Reply:
It's true that it's not super great here, but copy pasting this code in any app that needs it also isn't great. Also, I feel the API ship has sailed a little bit because now MusicVAE has 3 functions that require tensors (the new controls stuff), so we can't even pretend like we haven't exposed tensor implementation anymore

Reply:
It's not so much about the tensors but the fact that I added this composite function to the MusicVAE API, when maybe it should be in some downstream library.

Though I suppose interpolate is also a composite function...

Reply:
Yes, interpolate is composite but it's also a fairly standard use-case. This seems a bit more experimental. Also, couldn't you achieve the same result by combining a sample and interpolate?

Reply:
Not easily, because interpolate generates a fixed number of samples equally spaced between the endpoints.  You'd end up throwing most of them away.

Reply:
I don't have full context here, but "get similar" seems like a very standard use case at least in non-generative models (e.g. with just normal embedding spaces). I suspect this feature could get a lot of useage if integrated into musicVAE...